e2e/tests/submissions/author.test.js

```diff
@@ -53,38 +53,37 @@ describe('author submissions', () => {
     const workflowResponse = await page.goto(
       `${routes.HOLDINGPEN_API}/${submitResponseJson.workflow_object_id}`
     );
+
+    const expectedMetadata = {
```

Comment:
can you separate this into another commit? (or remove it, we can do it with all others later)